src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.FluentWindows/Styles/ProgressBar.xaml

```diff
@@ -0,0 +1,89 @@
+﻿<!--
+    This Source Code Form is subject to the terms of the MIT License.
+    If a copy of the MIT was not distributed with this file, You can obtain one at https://opensource.org/licenses/MIT.
+    Copyright (C) Leszek Pomianowski and WPF UI Contributors.
+    All Rights Reserved.
+-->
+
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <Style TargetType="{x:Type ProgressBar}">
+        <Setter Property="Foreground" Value="{DynamicResource ProgressBarForeground}" />
+        <Setter Property="Background" Value="{DynamicResource ProgressBarBorderBrush}" />
+        <Setter Property="BorderBrush" Value="{DynamicResource ProgressBarBackground}" />
```

Comment:
```suggestion
        <Setter Property="Background" Value="{DynamicResource ProgressBarBackground}" />
        <Setter Property="BorderBrush" Value="{DynamicResource ProgressBarBorderBrush}" />
```

Reply:
It was about getting the colors to each property correctly, so the overall purpose was being solved, but yes the naming should have been correct. Will go ahead and do the necessary changes.